content/docs/handling-events.md

```diff
@@ -1,72 +1,73 @@
 ---
 id: handling-events
-title: Handling Events
+title: Իրադարձությունների մշակում
 permalink: docs/handling-events.html
 prev: state-and-lifecycle.html
 next: conditional-rendering.html
 redirect_from:
   - "docs/events-ko-KR.html"
 ---
+Իրադարձությունների մշակումը React֊ի էլեմենտներում շատ նման է իրադարձությունների մշակմանը DOM էլեմենտներում։ Կան որոշ շարահյուսական տարբերություններ։
 
-Handling events with React elements is very similar to handling events on DOM elements. There are some syntactic differences:
+* React֊ում իրադարձությունների անունները փոքրատառի փոխարեն ուղտագիր են։
+* JSX֊ում դուք կարող եք վերագրել ֆունկցիան որպես իրադարձության մշակող, տողային ֆունկցիայի փոխարեն։
 
-* React events are named using camelCase, rather than lowercase.
-* With JSX you pass a function as the event handler, rather than a string.
-
-For example, the HTML:
+Օրինակ, HTML֊ը՝
 
 ```html
 <button onclick="activateLasers()">
-  Activate Lasers
+  Ակտիվացնել լազերնրենը
 </button>
 ```
 
-is slightly different in React:
+React-ում մի փոքր այլ է՝
 
 ```js{1}
 <button onClick={activateLasers}>
-  Activate Lasers
+    Ակտիվացնել լազերնրենը
 </button>
 ```
 
-Another difference is that you cannot return `false` to prevent default behavior in React. You must call `preventDefault` explicitly. For example, with plain HTML, to prevent the default link behavior of opening a new page, you can write:
+Մեկ այլ տարբերություններից է այն, որ React-ում դուք չեք կարող վերադարձնել `false`, որպեսզի կանխեք իսկզբանե նախատեսված գործողությունը։
```

Comment:
...You must call `preventDefault` <- էս նախադասությունը սկսելուց առաջ նոր տող չպիտի լինի։

Reply:
done